spec/API_specification/data_type_functions.md

```diff
@@ -4,9 +4,72 @@
 
 A conforming implementation of the array API standard must provide and support the following data type functions.
 
+
 <!-- NOTE: please keep the constants in alphabetical order -->
 
 ## Objects in API
+(function-broadcast-arrays)=
+### broadcast_arrays(*args, /)
+
+Broadcast any number of arrays against each other.
+
+#### Parameters
+
+-   ***args**: _\[&lt;array&gt;\]_
+
+    -   arrays to broadcast.
+
+#### Returns
+
+-   **out**: _\[&lt;array&gt;\]_
+
+    -   a list of arrays where more than one element of a broadcasted array may refer to a single memory location.
+
+
+(function-broadcast-to)=
+### broadcast_to(array, /, shape)
+
+Broadcast an array to a new shape.
+
+-   A `ValueError` exception must be raised if the array is not compatible with the new shape according  to the semantics defined in {ref}`broadcasting`.
+
+#### Parameters
+
+-   **array**: _&lt;array&gt;_
+
+    -   array to broadcast.
+
+-   **shape**: Tuple[int, …]
+
+    -   shape of the desired array.
+
+#### Returns
+
+-   **out**: _&lt;array&gt;_
+
+    -   an array with the given shape where more than one element may refer to a single memory location.
+
+(function-can-cast)=
+### can_cast(from_, /, *, to)
+
+Determines if a type conversion is allowed under the rules described by {ref}`type-promotion`.
+
+#### Parameters
+
+-   **from_**: _Union\[ &lt;dtype&gt, &lt;array&gt;]_
+
+    -   data type, scalar or array to cast from.
+
+-   **to**: _&lt;dtype&gt;_
```

Comment:
Not clear to me why NumPy, e.g., does not support both `dtype` and `array` here. This would allow

```python
result = can_cast(x1, x2)
```

where we can answer the question "Can `x1` be cast to the same dtype as `x2`?" without needing to explicitly type `x2.dtype`.

Reply:
I think we should add `array` in the spec, it makes sense to answer "Can `x1` be cast to the same dtype as `x2`?"